deploy/charts/templates/autoscale.yaml

```diff
@@ -22,4 +22,4 @@ spec:
         name: memory
         target:
           type: AverageValue
-          averageValue: 320Mi
+          averageValue: 150Mi
```

Comment:
Good call I'll work on that in a future PR. This would make it easier to experiment with different values across environments or even choose to run fewer/more instances...

So far the instances are pretty stable in their resource usage which is good to see!

Reply:
ref. https://github.com/kiva/ui/pull/1864